app/pages/ServiceListingPage/ServiceListingPage.tsx

```diff
@@ -174,7 +174,15 @@ export const ServiceListingPage = () => {
                 {resource.services
                   .filter((srv) => srv.id !== service.id)
                   .map((srv) => (
-                    <ServiceCard service={srv} key={srv.id} />
+                    <ServiceCard
+                      service={{
+                        ...srv,
+                        long_description: removeAsterisksAndHashes(
+                          srv.long_description
```

Comment:
I did this because the ServiceCard.spec.tsx test did not want the helper function inside the ServiceCard.tsx file and just trying to move on quickly